cmd/README.md

```diff
@@ -8,13 +8,13 @@ Command Line tool for managing the WSO2 Micro Integrator
 
 - ### Setting up the development environment
     1. Install [Go 1.12.x](https://golang.org/dl)
-    2. Fork the [repository](https://github.com/wso2/micro-integrator)
+    2. Fork the [repository](https://github.com/wso2/product-mi-tooling)
     3. Clone your fork into any directory
-    5. `cd` into cloned directory and then cd into `micro-integrator/cmd`
+    5. Navigate into cloned directory and then navigate into `product-mi-tooling/cmd`
```

Comment:
Navigate to the cloned directory and then to the `product-mi-tooling/cmd` directory

Reply:
fixed